app/src/main/res/layout/shoplist_row.xml

```diff
@@ -1,14 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/shopDeleteItem"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:paddingTop="0dp"
     android:paddingBottom="0dp">
 
     <CheckBox
         android:id="@+id/item_checkbox"
-        android:layout_width="0dp"
+        android:layout_width="308dp"
```

Comment:
using relative sizes might be better here as absolute ones wont change depending on the size of the phone

Reply:
I tried but could not get it to work. Maybe you could show me next week ?

Reply:
i just noticed this is in a linearLayout which makes it different than the ones i did, we can try and revamp that in another PR

Reply:
I finally found out how to do it so it should be good now ^^